contents/news/tips-on-how-to-prepare-a-talk/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Tips on preparing a talk
+date: 2015-03-20
+template: article.jade
+category: news
+---
+
+So you decided that you want to visit the JSunconf this year. As you might know a unconference lives from the visitors and he Talks are prepared by the visitors and get choosen in the morning of each conference day by the crowd. This gives everyone the chance to prepare an awesome talk. This is where you all come in.
+
+If you are terrified to speak in public or you have no idea hwo to prepare a talk you should have a look at [speaking.io](http://speaking.io/)
+
+If you work in a special environment, or you started an open source project or you have worked with a special library. It would be great if you can share your expirience with the audience. 
+What also apply's is if you have a deep knowledge of something (git, angular.js, react or pick any topic) it would be awesome if you can share your failures, experiences, learnings or conclusions.
+
+Anothere cool thing is if you have a great interesst in something and you might have accomplished to do some tutorials, and you are interessted in more information from people which already worked a longer period of time with that technologie you may organize a discussion group.
+
+But in any case this a D.I.Y. conference. And everybody should share something to the community. If you are scared to speak alone you may find a friend or colleague how can to a talk with you.
```

Comment:
`But in any case this a D.I.Y. conference` => `But in any case, this a D.I.Y. conference`